service/standalone_apis_test.go

```diff
@@ -59,10 +59,7 @@ func TestRevokeHandler(t *testing.T) {
 	})
 
 	t.Run("400 on missing resourceIds from body if resourceType request param is present", func(t *testing.T) {
-		reqBody := setupRevokeRequestBody(t, RevokeRequestBody{
-			Subjects: []string{"piero"},
-			Groups:   []string{"litfiba"},
-		})
+		reqBody := setupRevokeRequestBody(t, RevokeRequestBody{})
```

Comment:
can you add a test to verify 400 is received if everything is an empty list? does it make sense? (it does to me lol)

Reply:
Shouldn't it be already covered by this test? By sending `RevokeRequestBody{}` each array is deserialized as a `nil` pointer that from a `len` point of view is the same as and empty list. Am I wrong?

Reply:
yes, right!